lakefile.lean

```diff
@@ -21,7 +21,7 @@ lean_exe ssaLLVMEnumerator {
 
 /-- Separate lib since it takes quite a while to compile -/
 lean_lib AliveExamples {
-  roots := #[`SSA.Projects.InstCombine.AliveAutoGenerated]
+  roots := #[`SSA.Projects.InstCombine.AliveAutoGenerated].append #[`SSA.Projects.InstCombine.ScalingTest]
```

Comment:
Why do you have the `append`, intead of:
```suggestion
  roots := #[`SSA.Projects.InstCombine.AliveAutoGenerated, `SSA.Projects.InstCombine.ScalingTest]
```

I think I'd like the `ScalingTest` to be its own compilation target as well, since I'd rather not have to run it everytime I'd like to test improvements to the Alive automation

Reply:
The second did not work for me for some reason. I guess I missed the tick before the 2nd SSA. I introduced now a new Target AliveScaling, which should avoid this issue alltogether.